blocks/media-upload-button/index.js

```diff
@@ -77,13 +34,9 @@ class MediaUploadButton extends Component {
 			frameConfig.library = { type };
 		}
 
-		if ( gallery ) {
-			const GalleryDetailsMediaFrame = getGalleryDetailsMediaFrame();
-			this.frame = new GalleryDetailsMediaFrame( {
-				frame: 'select',
-				mimeType: type,
```

Comment:
One thing that is lost here is the ability to specify non-images as the type to select. So that would prevent this from being able to be used for making audio or video playlists, in other words “non-image galleries”.